util.go

```diff
@@ -292,3 +293,30 @@ func contains[T comparable](s []T, e T) bool {
 func chooseRandomFromRange(min float64, max float64) float64 {
 	return rand.Float64()*(max-min) + min
 }
+
+func isDbusDaemonRunning() bool {
+	// TODO: delete this once we replaced 99designs/keyring (SNOW-1017659) and/or keyring#103 is resolved
+	cmd := exec.Command("pidof", "dbus-daemon")
+	_, err := cmd.Output()
+
+	// false: process not running, pidof not available (sysvinit-tools, busybox, etc missing)
+	return err == nil
+}
+
+func canDbusLeakProcesses() (bool, string) {
+	// TODO: delete this once we replaced 99designs/keyring (SNOW-1017659) and/or keyring#103 is resolved
+	leak := false
+	message := ""
+
+	valDbus, haveDbus := os.LookupEnv("DBUS_SESSION_BUS_ADDRESS")
+	if !haveDbus || strings.Contains(valDbus, "unix:abstract") {
+		// if DBUS_SESSION_BUS_ADDRESS is not set or set to an abstract socket, it's not necessarily a problem, only
```

Comment:
Is there some comment missing?

Reply:
no, i was considering lines 313-314 together 😅  , comment meaning ... 'not necessarily a problem, only if the dbus-daemon is running'

Reply:
I'd prefer having full sentences, not joining comments with code :)

Reply:
fixed!